lms/djangoapps/verify_student/tests/factories.py

```diff
@@ -6,10 +6,13 @@
 from verify_student.models import SoftwareSecurePhotoVerification
 
 
+# Factories are self documenting
+# pylint: disable=missing-docstring
```

Comment:
Do we need this here? Since the class has a docstring. Also, instead of putting in exceptions for the pylint checker, it would be cleaner to just add one line docstrings to the classes.